components/eam/bld/namelist_files/namelist_defaults_eam.xml

```diff
@@ -285,6 +285,9 @@
 <mam4_mode2_file rad="rrtmg">atm/cam/physprops/mam4_mode2_rrtmg_c130628.nc</mam4_mode2_file>
 <mam4_mode3_file rad="rrtmg">atm/cam/physprops/mam4_mode3_rrtmg_aeronetdust_c141106.nc</mam4_mode3_file>
 <mam4_mode4_file rad="rrtmg">atm/cam/physprops/mam4_mode4_rrtmg_c130628.nc</mam4_mode4_file>
+<!-- hybrown -->
+<mam4_mode1_file rad="rrtmg" cldera_strat_volc="1">/projects/cldera/data/E3SM/model_input/modal_optics/modal_optics_mode1_hb.nc</mam4_mode1_file>
```

Comment:
This is hardcoded to SNL and won't work on NERSC. Not sure what our workaround should be, given that the project spaces are in different places on different machines.